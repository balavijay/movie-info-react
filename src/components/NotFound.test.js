import React from 'react';
import { shallow } from 'enzyme';

import NotFound from './NotFound';

describe ("NotFound Component " , () => {
   const  app = shallow(<NotFound />);
    test("Component rendered" , () => {
        expect(app).toMatchSnapshot();
    })
});