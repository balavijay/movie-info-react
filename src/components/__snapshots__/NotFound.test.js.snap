// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound Component  Component rendered 1`] = `ShallowWrapper {}`;
